src/node.rs

```diff
@@ -128,8 +128,6 @@ pub(crate) struct OutlineNode {
 }
 
 impl OutlineNode {
-    pub(crate) const IN_VIEW: &'static str = "view";
```

Comment:
these changes are again due to this

https://bevyengine.org/learn/migration-guides/0.10-0.11/#make-render-graph-slots-optional-for-most-cases